.aspell.en.pws

```diff
@@ -61,4 +61,7 @@ RBAC
 YAML
 DNS
 hostname
+collectinfo
```

Comment:
Generally these should be wrapped in `collectinfo` to exclude.

Reply:
@spjmurray is doing a docs guide as we speak on this.

Reply:
I think this got caught because it's in the link syntax - that's the only place that `collectinfo` is mentioned in the asciidoc:

```
link:/collectinfo.html[Collect Information]
```

Reply:
Ok, rename to just collect then or maybe hyphenated. You should be able to prevent the failure.